### ISSUE TITLE
Test data for dogs^3

### ISSUE THREAD

**Issue by ryzhyk** [TimelyDataflow/differential-dataflow#182]
Any chance you could add links to test data or include smaller data files for some of the examples in `dogsdogsdogs` in the repo?

Thanks!

**Comment by top1select** [comment]
Yes, please add some example data files, otherwise it's really hard to follow your examples.
